src/main/java/Event.java

```diff
@@ -0,0 +1,41 @@
+package ip.derrick ;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task{
+
+    String start;
+    String end;
+    LocalDate newStart;
```

Comment:
Could newStart give a different impression of what this variable represents, would formattedStart be better?